src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PAY_PER_DAY = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+
+  if (days < SHORT_TERM) {
+    return days * PAY_PER_DAY;
+  }
+
+  if (days >= LONG_TERM) {
+    return days * PAY_PER_DAY - BIG_DISCOUNT;
+  }
```

Comment:
The condition `if (days >= LONG_TERM)` is correct, but since you are returning inside the `if` block, you don't need to check `if (days >= SHORT_TERM)` again in the subsequent condition. You can simply return the calculation for the small discount after the first `if` block, as any number of days that is not long term but is more than or equal to short term will fall into this case.